users.go

```diff
@@ -437,3 +437,152 @@ func (s *UsersService) UnblockUser(user int) error {
 		return fmt.Errorf("Received unexpected result code: %d", resp.StatusCode)
 	}
 }
+
+// EMail represents an EMail.
+//
+// GitLab API docs: https://doc.gitlab.com//help/api/users.md#list-emails
```

Comment:
This URL is not correct, should be: `http://doc.gitlab.com/ce/api/users.html#list-emails`